src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowCrudV2Spec.groovy

```diff
@@ -1138,7 +1138,7 @@ class FlowCrudV2Spec extends HealthCheckSpecification {
 
         and: "Update the flow: vlanId on the src endpoint"
         def updatedFlowSrcEndpoint = flow.jacksonCopy().tap {
-            it.source.vlanId = flow.destination.vlanId + 1
+            it.source.vlanId = flow.destination.vlanId - 1
```

Comment:
out of curiosity: why `-` ?

Reply:
we pick vlans  for a flow in range `100..4095`. When 4095 is picked, a 4096 update appears as an invalid operation (allowed values are 0..4095)